api/net/Middleware/ErrorHandlingMiddleware.cs

```diff
@@ -95,7 +95,9 @@ private async Task HandleExceptionAsync(HttpContext context, Exception ex)
                 code = HttpStatusCode.BadRequest;
                 message = "Data may have been modified or deleted since item was loaded.  Refresh your data and reapply your changes.";
 
-                _logger.LogDebug(ex, "Optimistic concurrency error", ex.Message);
+                _logger.LogError(ex,
+                    "Optimistic concurrency error (user agent: {userAgent})",
+                    context.Request.Headers["User-Agent"].ToString());
```

Comment:
You removed the `ex.Message`.  Any reason?

Reply:
Removing it as ex.Message is redundant here and not actually being used based on the LogError method's signature.